functions/palindrome_number.py

```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+  temp = x
+  reverse = 0
+  while(x>0):
+    dig = x%10
+    reverse = reverse*10 + dig
+    x = x//10
+
+    if temp == reverse:
+      return True
+    else:
+      return False
```

Comment:
hey, Amel! I think the test is failing because it is returning False while the assert checking with true